app/models/topics.models.js

```diff
@@ -0,0 +1,5 @@
+const db = require("../../db/connection");
+
+exports.selectAllTopics = () => {
+	return db.query(`SELECT * FROM topics`).then(({ rows }) => rows);
+};
```

Comment:
Great stuff - nice use of destructuring throughout to keep your code succinct and readable